tests/Unit/GitlabActionsCommandTest.php

```diff
@@ -41,6 +52,34 @@ public function testExecute() {
       ->assertOutputContains('"project_id": 12345');
   }
 
+  public function testMissingVars() {
```

Comment:
Maybe a small DocBlock would be great here (for CS).

Reply:
fixed